samples/client/petstore/swift5/nonPublicApi/PetstoreClient/Classes/OpenAPIs/Models/PropertyNameMapping.swift

```diff
@@ -0,0 +1,48 @@
+//
+// PropertyNameMapping.swift
+//
+// Generated by openapi-generator
+// https://openapi-generator.tech
+//
+
+import Foundation
+#if canImport(AnyCodable)
+import AnyCodable
+#endif
+
+internal struct PropertyNameMapping: Codable, JSONEncodable, Hashable {
+
+    internal var httpDebugOperation: String?
+    internal var type: String?
+    internal var type: String?
+    internal var type: String?
+    internal var type: String?
```

Comment:
@wing328 all the properties have the same name, this won't compile

Reply:
yes, this won't compile without using the mapping option, e.g.
```yaml
generatorName: swift5
outputDir: samples/client/petstore/swift5/nonPublicApi
inputSpec: modules/openapi-generator/src/test/resources/2_0/swift/petstore-with-fake-endpoints-models-for-testing.yaml
templateDir: modules/openapi-generator/src/main/resources/swift5
generateAliasAsModel: true
additionalProperties:
  podAuthors: ""
  podSummary: PetstoreClient
  sortParamsByRequiredFlag: false
  nonPublicApi: true
  projectName: PetstoreClient
  podHomepage: https://github.com/openapitools/openapi-generator
nameMappings:
  _type: underscoreType
  type_: typeWithUnderscore
  -type: dashType
parameterNameMappings:
  _type: underscoreType
  type_: typeWithUnderscore
  -type: dashType
```

I removed the files as these files are not supposed to be included. You can refer to the diff in the description to understand the change before and after using nameMappings, parameterNameMappings in the config.

Reply:
And this is what the schema looks like that will result in naming collisions:
```
components:
  schemas:
    PropertyNameMapping:
      properties:
        http_debug_operation:
          type: string
        _type:
          type: string
        type:
          type: string
        type_:
          type: string
        -type:
          type: string

```